Converted to include the "add HTML" component

NicMcPhee opened the pull request:
This adds a new component to the lab where they add some HTML that lists some tools that we're working with in this class. They then need to provide some styling for that new section, and only make *one* new CSS style for the whole page.

Thanks to @floogulinc for the encouragement to have HTML play a larger role in this lab, and to @kklamberty for coming up with the idea of having the new HTML section be about tools that we're using in this course.

It would be good to have someone proofread both `README.md` and `LABTASKS.md` both in general, but also with a particular eye for places where I may have missed removing a reference to creating *two* new style files.

NicMcPhee committed:
Converted to include the "add HTML" component

This adds a new component to the lab where they add some HTML that lists some tools that we're working with in this class. They then need to provide some styling for that new section, and only make *one* new CSS style for the whole page.

NicMcPhee committed:
Remove `style3.css`

Since they're only doing one additional styling now, I figured we really should remove this other style file.

NicMcPhee committed:
Remove reference to third style file in the HTML

Now that the third style file isn't there, I figured we should remove the link to it in the navbar.

NicMcPhee committed:
Clarified a bit of LABTASKS

I honestly am not 100% sure I know what was intended by the "Our Team's GitHub Pages URL" section, but I added some text to hopefully clarify what I _thought_ was going on there.

kklamberty commented on LABTASKS.md:
```diff
@@ -1,25 +1,100 @@
+  the HTML.
+
+All of the formatting (including the positioning of elements in the page) must
+  be done using CSS. **Do not alter the HTML** with two exception:
```
'two exception' should be 'two exceptions'

kklamberty commented on LABTASKS.md:
```diff
@@ -1,25 +1,100 @@
+that provides a list of some of the tools we'll be using in this class:
+
+- [`git`](https://git-scm.com/)
+- GitHub.com
```
Shouldn't this have a link like the other items in the list?

NicMcPhee replied:
Good eye. Turns out that GitHub sees this as a URL and automatically turns it into a link when it renders it. Could add an explicit link for consistency, though, if you'd like.

kklamberty replied:
I don't think it's important to make it explicit as long as it renders as a URL so they know what URL to use in their html link.

kklamberty replied:
When I view labtasks in that branch on GitHub, it does not seem to render it as a link.

floogulinc replied:
While it does convert to a link in VS Code's preview, it does not appear to on GitHub.

NicMcPhee replied:
Good catch. I'd assumed that since it was a link in VS Code it would be a link on GitHub. Thanks for fixing that!

kklamberty commented on LABTASKS.md:
```diff
@@ -1,25 +1,100 @@
+Add CSS that incorporates this new HTML into the look and feel of the original
+version of the web page. A few specific requirements:
+
+- It should be displayed as a block like the other three sections.
```
'It' should maybe be 'The tool list information' or something - just to clarify that it's not the CSS that is displayed as a block. It's pretty clear, but sometimes pronouns can be confusing.

kklamberty commented on LABTASKS.md:
```diff
@@ -1,25 +1,100 @@
+
+- It should be displayed as a block like the other three sections.
+- The header should be obviously different from the content.
+- The `local` and `cloud` items in the list should be formatting differently.
```
'formatting' should be 'formatted'

kklamberty replied:
In lines 83 and 84

kklamberty reviewed (changes requested):
I pointed out a couple of minor typos. I was reading this directly in GitHub, so I did not check that the markdown is used consistently or view a preview of the text, but I did look at all the changes in the 4 commits of the pull request.

floogulinc committed:
Fix some typos and issues in LABTASKS.md

floogulinc commented:
I have made the changes @kklamberty requested

floogulinc commented on LABTASKS.md:
````diff
@@ -1,25 +1,100 @@
+You can provide multiple, space-separated classes for an item, e.g.,
+
+```html
+    <li class="local version-control" id="git"><code>git</code></li>
````
This implies that all the elements in the list should be in code blocks, maybe provide two examples here like:
```html
<li class="local version-control" id="git"><a href="https://git-scm.com/"><code>git</code></a></li>
<li class="local ide" id="vscode"><a href="https://code.visualstudio.com/">Visual Studio Code</a></li>
```

Also I noticed that the examples aren't links, so I added that to my example. They probably should be since that is requested.

NicMcPhee replied:
Good catch – thanks!

NicMcPhee replied:
Dealt with this in 726bff4.

floogulinc committed:
Replace IDEA instructions with VS Code

NicMcPhee committed:
Extend/complete the multiple class example

@floogulinc suggested some good changes to the example of having multiple classes on an HTML element, which this incorporates.

NicMcPhee reviewed (commented):
I think this all looks good now. Thanks to Paul for addressing KK's changes and suggesting some additional improvements. I dealt with the suggestion about changes to the multiple class example and URLs in 726bff4.

NicMcPhee commented:
I'm going to go ahead and merge this in now. Thanks to everyone for the help and feedback!
